LEARNER-1940 - Restructure i18n files to ensure they're available in Django app

amangano-edx opened the pull request:
This PR restructures the i18n/locale files in credentials-themes to follow Django convention so that they may be used by the credentials service.

According to https://docs.djangoproject.com/en/1.11/topics/i18n/translation/#how-django-discovers-translations, Django discovers translation files (.po/.mo files) by first looking in the directories configured via the LOCALE_PATHS setting, then by looking for a 'locale' directory in each installed application. The edx_credentials_themes directory is an installed application for the credentials service, therefore Django will expect any translations for the themes to live in 'edx_credentials_themes/locale'. 

JIRA: https://openedx.atlassian.net/browse/LEARNER-1940

clintonb commented on Makefile:
```diff
@@ -8,13 +8,13 @@ build.watch:
 
 i18n_requirements:
-	pip install -r ./conf/locale/requirements.txt
+	pip install -r ./i18n/requirements.txt
```
Move this file to requirements/i18n.txt.

clintonb commented on edx_credentials_themes/__init__.py:
```diff
@@ -1 +1 @@
-__version__ = '0.1.4'
+__version__ = '0.1.6'
```
0.1.5

amangano-edx replied:
0.1.5 seems to already exist (there's a tag for it). I think someone may have just forgotten to bump the version.

amangano-edx commented:
@clintonb I've addressed your feedback, and also added an entry to the MANIFEST.in file to ensure that the .mo files are included in the package when it's installed. When you have a second, could you please take one more look?

Also, post merge, do we typically create tags for each version bump?

BenjiLee commented:
@amangano-edx I believe you also need to create a new release on this github repo after this merges.
